opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -221,6 +260,9 @@ impl Logger {
         instrumentation_lib: Arc<InstrumentationLibrary>,
         provider: LoggerProvider,
     ) -> Self {
+        otel_info!(
+            name: "logger_new"
```

Comment:
We should probably add some identification information for the logger such as name, version etc.